app/assets/stylesheets/components/_cards.scss

```diff
@@ -8,12 +8,9 @@
   background-position: center;
   background-size: cover;
   min-width: 0;
-
-  @media screen and (min-width: $small) {
-    width: 250px;
-    height: 125px;
-    font-size: 20px;
-  }
+  width: 250px;
+  height: 125px;
+  font-size: 20px;
```

Comment:
If you're gonna set the value the same as the media query then you can just remove the media query.
Then it will be 250px form 0 to $large then it becomes 200px

Reply:
Nevermind I missread